packages/application-shell/src/components/fetch-user/fetch-user.spec.js

```diff
@@ -1,13 +1,36 @@
+import { graphql } from 'msw';
+import { setupServer } from 'msw/node';
 import React from 'react';
-import { GRAPHQL_TARGETS } from '@commercetools-frontend/constants';
-import { renderApp } from '../../test-utils';
-import LoggedInUserQuery from './fetch-user.mc.graphql';
+import { reportErrorToSentry } from '@commercetools-frontend/sentry';
+import { UserMock } from '../../../../../graphql-test-utils';
+import {
+  experimentalRenderAppWithRedux,
+  screen,
+  waitForElementToBeRemoved,
+} from '../../test-utils';
 import FetchUser from './fetch-user';
 
 jest.mock('@commercetools-frontend/sentry');
 
+const mockServer = setupServer(
```

Comment:
I was wondering if we should expose a msw helper/package library from app-kit which allows you to easily mock things also with variables and/or the target.

For now I think this is great but if we would have `msw.withVariables({ target}).withQuery('FetchLoggedInUser').respond(...)` it might keep setup a bit learner? WDYT?

Reply:
We can do something yes but I wouldn't worry about that now. I would first migrate all tests, then see what we can simplify setup-wise out of that.

What we can start thinking about though, is how we can expose/reuse our internal test data.

Reply:
Totally agree. Just thinking/tinkering forward.